GDJS/GDJS/IDE/ExporterHelper.cpp

```diff
@@ -797,6 +799,37 @@ bool ExporterHelper::ExportIncludesAndLibs(
   return true;
 }
 
+void ExporterHelper::ExportFreeFunctionIncludes(
+    gd::Project &project, std::vector<gd::String> &includesFiles) {
+  auto addIncludeFiles = [&](const std::vector<gd::String> &newIncludeFiles) {
+    for (const auto &includeFile : newIncludeFiles) {
+      InsertUnique(includesFiles, includeFile);
+    }
+  };
+
+  auto usedExtensions = gd::UsedExtensionsFinder::ScanProject(project);
```

Comment:
There is `auto usedExtensions = gd::UsedExtensionsFinder::ScanProject(project);` called already in `ExportWholePixiProject`. To avoid doing it twice (and to avoid encouraging traversing the project too much, which can become costly at the project becomes larger), I think we can pass these as parameter. You just need also put this in `ExportProjectForPixiPreview`, which is fine.